src/hydra-queue-runner/nar-extractor.cc

```diff
@@ -0,0 +1,82 @@
+#include "nar-extractor.hh"
+
+#include "archive.hh"
+
+#include <unordered_set>
+
+using namespace nix;
+
+struct Extractor : ParseSink
+{
+    std::unordered_set<Path> filesToKeep {
+        "/nix-support/hydra-build-products",
+        "/nix-support/hydra-release-name",
+        "/nix-support/hydra-metrics",
+    };
```

Comment:
This is for the various functions that intake data from the build result and store in the db, right? Is this going to change/break other behavior?

Reply:
It shouldn't change any behaviour.